packages/core/src/geometry/Attribute.js

```diff
@@ -8,7 +8,7 @@
  * @class
  * @memberof PIXI
  */
-class Attribute
+export default class Attribute
```

Comment:
I think these exports should be named exports, IIRC we decided to not do export default whenever possible.